jumpscale/sals/vdc/size.py

```diff
@@ -15,6 +15,11 @@
 
 INITIAL_RESERVATION_DURATION = 1  # in hours
 
+ETCD_CPU = 1
+ETCD_MEMORY = 2 * 1024  # in MB
```

Comment:
can it be lower?

Reply:
I don't know the prod requirements for the cluster for k8s deployment. but it can be changed for sure